pandas/tests/series/test_logical_ops.py

```diff
@@ -513,3 +513,21 @@ def test_int_dtype_different_index_not_bool(self):
 
         result = ser1 ^ ser2
         tm.assert_series_equal(result, expected)
+
+    def test_any_all(self):
```

Comment:
It seems there is no other test for any/all in this file. Maybe `pandas/tests/reductions/test_reductions.py` is a better place to put them (or in a file specific for the string arrays)

Reply:
moved